types/braintree-web/test/web-new3ds.ts

```diff
@@ -0,0 +1,794 @@
+import * as braintree from 'braintree-web';
```

Comment:
What is the difference between this new test file and the current [`web.ts`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/braintree-web/test/web.ts) test file? It looks like there's a lot of duplication between them. Could you just add your new test to that file?

Reply:
yes sure, will remove the redundant parts

Reply:
All right, done